terraform/7-rds.tf

```diff
@@ -0,0 +1,94 @@
+module "db" {
+  source  = "terraform-aws-modules/rds/aws"
+
+  identifier = "libera-postgresql"
+
+  # All available versions: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts
+  engine               = "postgres"
+  engine_version       = "14"
+  family               = "postgres14" # DB parameter group
+  major_engine_version = "14"         # DB option group
+  instance_class       = "db.t4g.large"
+
+  allocated_storage     = 20
+  max_allocated_storage = 100
+
+  create_db_option_group = true
+  create_db_parameter_group = true
+
+
+  # NOTE: Do NOT use 'user' as the value for 'username' as it throws:
+  # "Error creating DB Instance: InvalidParameterValue: MasterUsername
+  # user cannot be used as it is a reserved word used by the engine"
+  db_name  = "liberadbname"
+  username = "liberadbusername"
+  port     = 5432
```

Comment:
esta bien que este harcodeado asi el puerto?

Reply:
Nope, it is not necessary